// Copyright (c) 2009 AdCopy
// Author: Jeff Weisberg
// Created: 2009-May-12 16:41 (EDT)
// Function: 
//
// $Id$

message ACPScriblRequest {
        required string         filename        = 1;
        optional string         hash_sha1       = 2;
}

message ACPScriblReply {
	required int32		status_code	= 1;
	optional string		status_message	= 2;
        optional string         hash_sha1       = 3;
}

